packages/fondue/src/components/TextInput/TextInput.stories.tsx

```diff
@@ -152,7 +162,7 @@ export const FocusOnMount = TextInputTemplate.bind({});
 
 FocusOnMount.args = {
     value: 'Value text',
-    focusOnMount: true,
+    focus: true,
```

Comment:
I think since we've deprecated the `focusOnMount` from the API, we can remove this Story altogether, or not make changes to the current Storybook.

The way to setup focus programmatically is pretty standard and not a foundue-specific implementation, so we don't need to document it further

Reply:
Removed the story